.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+yarn.lock
+package-lock.json
```

Comment:
are we sure about these 2?

Reply:
see [this comment](https://github.com/wilsoniovlabs/relaying-services-sdk-dapp/pull/1#issuecomment-961190809) too